pkg/reconciler/kubernetes/tektonpipeline/tektonpipeline.go

```diff
@@ -363,8 +363,19 @@ func (r *Reconciler) targetNamespaceCheck(ctx context.Context, tp *v1alpha1.Tekt
 	ns, err := r.kubeClientSet.CoreV1().Namespaces().Get(ctx, tp.GetSpec().GetTargetNamespace(), metav1.GetOptions{})
 	if err != nil {
 		if apierrors.IsNotFound(err) {
+			if err := common.CreateOperatorVersionConfigMap(r.manifest, tp); err != nil {
+				return err
+			}
+
+			if err := r.createTargetNamespace(ctx, labels, tp); err != nil {
+				return err
+			}
```

Comment:
```suggestion
			if err := common.CreateOperatorVersionConfigMap(r.manifest, tp); err != nil {
				return err
			}

			if err := r.createTargetNamespace(ctx, labels, tp); err != nil {
				return err
			}
```
```suggestion

                         if err := r.createTargetNamespace(ctx, labels, tp); err != nil {
				return err
			}
			if err := common.CreateOperatorVersionConfigMap(r.manifest, tp); err != nil {
				return err
			}

			
```

Logically, the namespace should be created before we can create the configmap. 🧑‍💻